cost-analyzer/templates/query-service-cluster-role-binding-template.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and (not .Values.agent) (not .Values.cloudAgent) (.Values.kubecostDeployment) }}
+{{- if and (.Values.kubecostDeployment.queryServiceReplicas) (.Values.kubecostDeployment.queryServiceReplicas.enabled) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "query-service.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "query-service.commonLabels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "query-service.serviceAccountName" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "query-service.serviceAccountName" . }}
+    namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "query-service.serviceAccountName" . }}
+  labels:
+    {{ include "query-service.commonLabels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "query-service.serviceAccountName" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ template "query-service.serviceAccountName" . }}
+    namespace: {{ .Release.Namespace }} 
+{{- end }}
+{{- end }}
```

Comment:
Why do we have a `ClusterRole`(`Binding`) for query service? What access does it need to resources outside of the release namespace?

Reply:
Well that would be because I still only have a beginner's understanding of roles and such, and misunderstood what this does. I'll take it out, as we don't need to access anything outside of the release namespace.

Reply:
@nik-kc looks like we still have a clusterRoleBinding here?

Reply:
Fixed, thanks for noticing that.